docs/modules/configure/pages/environment_variables.adoc

```diff
@@ -63,6 +63,12 @@ We try to let most of the things defined through Environment Variables, as https
 |SMTP_USERNAME
 |Username for the SMTP server, for sending emails. See xref:services:smtp.adoc[SMTP]
 
+|VAPID_PUBLIC_KEY
+|VAPID public key that will be used to sign the Push API requests. It can be generated runing the task: `rake decidim:generate`
```

Comment:
```suggestion
|VAPID public key that will be used to sign the Push API requests. It can be generated runing the task: `rake decidim:generate_vapid_keys`
```